src/api/registration/registration.service.js

```diff
@@ -68,7 +72,7 @@ const saveRegistration = async (registration, fsa_rn, council) => {
     }
 
     const metadata = await createMetadata(registration.metadata, reg.id);
-
+    updateStatusInCache(fsa_rn, "registration", true);
```

Comment:
This is causing jest to exit with open handles. Async calls like this must be mocked out.